### DISCUSSION TITLE
Settings Expander not working

### DISCUSSION THREAD

**Discussion by DavidM29** [unoplatform/uno#17466] [Q&A] [answered]
SettingsExpander is not visible on android:
![image](https://github.com/unoplatform/uno/assets/79500334/29f4e1fc-0b4a-430f-bef0-c35ddde16bd9)

I was not sure where to post this, here or in windows community toolkit, that is why I opened the discussion.

**Comment by DavidM29** [comment]
Here is the xaml:
`<Page
    x:Class="BusinessAssistantPosUno.Views.SettingsPage"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:controls="using:CommunityToolkit.WinUI.Controls"
    xmlns:converters="using:CommunityToolkit.WinUI.Converters"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    xmlns:me="using:BusinessAssistantPosUno.MarkupExtensions"
    mc:Ignorable="d">

    <Page.Resources>
        <converters:BoolToVisibilityConverter x:Key="BoolToVisibilityConverter" />
    </Page.Resources>

    <Grid RowDefinitions="Auto,*,Auto" Style="{StaticResource PageContainerGrid}">

        
        <Grid Grid.Row="0">
            <TextBlock Style="{ThemeResource TitleTextBlockStyle}" Text="{me:StringResource Key=Settings}" />
        </Grid>

        
        <ScrollViewer Grid.Row="1">
            <StackPanel x:DefaultBindMode="TwoWay" Spacing="4">

                
                <controls:SettingsExpander>
                    <controls:SettingsExpander.Items>
                        <controls:SettingsCard
                            HorizontalContentAlignment="Stretch"
                            ContentAlignment="Vertical"
                            Header="{me:StringResource Key=ODataServiceEndpoint}">
                            <TextBox Text="{x:Bind ViewModel.PosConfig.ODataServiceEndpoint, FallbackValue={x:Null}}" />
                        </controls:SettingsCard>
                        <controls:SettingsCard
                            HorizontalContentAlignment="Stretch"
                            ContentAlignment="Vertical"
                            Header="{me:StringResource Key=ODataServiceUsername}">
                            <TextBox Text="{x:Bind ViewModel.PosConfig.ODataServiceUsername, FallbackValue={x:Null}}" />
                        </controls:SettingsCard>
                        <controls:SettingsCard
                            HorizontalContentAlignment="Stretch"
                            ContentAlignment="Vertical"
                            Header="{me:StringResource Key=ODataServicePassword}">
                            <PasswordBox Password="{x:Bind ViewModel.PosConfig.ODataServicePassword, FallbackValue={x:Null}}" />
                        </controls:SettingsCard>
                        <controls:SettingsCard
                            HorizontalContentAlignment="Stretch"
                            ContentAlignment="Vertical"
                            Header="{me:StringResource Key=PosSetupServiceName}">
                            <TextBox Text="{x:Bind ViewModel.PosConfig.PosSetupServiceName, FallbackValue={x:Null}}" />
                        </controls:SettingsCard>
                        <controls:SettingsCard
                            HorizontalContentAlignment="Stretch"
                            ContentAlignment="Vertical"
                            Header="{me:StringResource Key=CompanyName}">
                            <TextBox Text="{x:Bind ViewModel.PosConfig.CompanyName, FallbackValue={x:Null}}" />
                        </controls:SettingsCard>
                    </controls:SettingsExpander.Items>
                </controls:SettingsExpander>

                
                <controls:SettingsExpander
                    Header="{me:StringResource Key=UserConfig}"
                    IsExpanded="True"
                    Visibility="{x:Bind ViewModel.IsUserContentVisible, Converter={StaticResource BoolToVisibilityConverter}}">
                    <controls:SettingsExpander.HeaderIcon>
                        <FontIcon Glyph="&#xE77B;" />
                    </controls:SettingsExpander.HeaderIcon>
                    <controls:SettingsExpander.Items>
                        <controls:SettingsCard Content="{x:Bind ViewModel.PosUser.DefaultCustomerNo, FallbackValue={x:Null}}" Header="{me:StringResource Key=DefaultCustomerNo}" />
                        <controls:SettingsCard Header="{me:StringResource Key=BankPaymentEnabled}">
                            <CheckBox
                                MinWidth="0"
                                Padding="0"
                                IsChecked="{x:Bind ViewModel.PosUser.BankPaymentEnabled, FallbackValue=False}"
                                IsHitTestVisible="False" />
                        </controls:SettingsCard>
                    </controls:SettingsExpander.Items>
                </controls:SettingsExpander>

                
                <controls:SettingsCard Header="{me:StringResource Key=Language}">
                    <controls:SettingsCard.HeaderIcon>
                        <FontIcon Glyph="&#xF2B7;" />
                    </controls:SettingsCard.HeaderIcon>
                    <ComboBox ItemsSource="{x:Bind ViewModel.SupportedLanguages, Mode=OneWay}" SelectedItem="{x:Bind ViewModel.PosConfig.Language, FallbackValue={x:Null}}" />
                </controls:SettingsCard>

                
                <controls:SettingsCard Header="{me:StringResource Key=SignOut}" Visibility="{x:Bind ViewModel.IsUserContentVisible, Converter={StaticResource BoolToVisibilityConverter}}">
                    <controls:SettingsCard.HeaderIcon>
                        <FontIcon Glyph="&#xF3B1;" />
                    </controls:SettingsCard.HeaderIcon>
                    <Button Command="{x:Bind ViewModel.SignOutCommand, Mode=OneWay}">
                        <FontIcon Foreground="Red" Glyph="&#xF3B1;" />
                    </Button>
                </controls:SettingsCard>
            </StackPanel>
        </ScrollViewer>

        
        <Grid
            Grid.Row="2"
            HorizontalAlignment="Right"
            ColumnSpacing="8">

            <Grid.ColumnDefinitions>
                <ColumnDefinition />
                <ColumnDefinition />
            </Grid.ColumnDefinitions>

            <Button
                Grid.Column="0"
                Width="100"
                Command="{x:Bind ViewModel.SaveCommand}"
                Content="{me:StringResource Key=Save}" />

            <Button
                Grid.Column="1"
                Width="100"
                Command="{x:Bind ViewModel.CancelCommand}"
                Content="{me:StringResource Key=Cancel}" />

        </Grid>
    </Grid>
</Page>
`

**Reply by jeromelaban** [reply, marked as answer]
This is being tracked here: https://github.com/unoplatform/uno/issues/16195 /cc @MartinZikmund
